build_openstudio_gems.rb

```diff
@@ -97,7 +100,38 @@ def make_package(install_dir, tar_exe, expected_ruby_version, bundler_version)
   system_call("#{bundle_cmd} config")
   puts "=" * 80
 
-  system_call("#{bundle_cmd} install")
+  puts "=" * 16 + " precaching packages and building them manually " + "=" * 16
+  # Shenigans onto shenanigans: Bundler, when you specify :git or :path
+  # actually installs in #{install_dir}/ruby/#{ruby_gem_dir} under the
+  # bundler/gems folder, and not under the gems/ one
+  # and that's causing a lot of troubles for openstudio to find them
+  # So here we:
+  #   * Download everything into the vendor/cache folder
+  #   * Find everything that's a directory, not a .gem file, go in there, build
+  #   it and move it back up
+  #   * Then we bundle install --local to use that vendor/cache,
+  #   and all is _well_
```

Comment:
Read this @kbenne

Reply:
Thanks. This is quite the gynastics.  

Here's a specific example of what I was seeing and now should be resolved thanks to your change.

The embedded path to the gem and sepcifically the gemspec is
 :/ruby/3.2.0/bundler/gems/tbd-ad400ccc5903/tbd.gemspec,
but
The include path that is injected by the rubygems gears is :/ruby/3.2.0/bundler/gems/gems/tbd-3.4.0/lib

Specification::activate apparently edits LOAD_PATH, but for whatever reason it injects the latter not the former path, and that is wrong.